mne_icalabel/tests/test_features.py

```diff
@@ -118,6 +123,30 @@ def test_compute_ica_activations():
     assert np.allclose(icaact, icaact_eeglab, atol=1e-4)
 
 
+# ----------------------------------------------------------------------------
+def test_loc():
```

Comment:
I'm not sure I 100% follow the use of this test.

Is it to ensure that MNE doesn't break anything when loading in EEG coordinates stored from EEGLab?

Reply:
It's mostly a sanity check. If you load the `.set` in EEGLAB and in MNE, the electrode locations are read in the appropriate format for each. Thus, on the MNE side, if we read the `.set` file and convert the montage back to EEGLAB format with `mne_to_eeglab_locs`, the resulting `rd` and `th` have to match the location retrieved in EEGLAB.